flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/io/CheckpointBarrierUnaligner.java

```diff
@@ -154,46 +154,51 @@ public void processBarrier(CheckpointBarrier receivedBarrier, int channelIndex)
 	@Override
 	public void processCancellationBarrier(CancelCheckpointMarker cancelBarrier) throws Exception {
 		long cancelledId = cancelBarrier.getCheckpointId();
-		if (LOG.isDebugEnabled()) {
-			LOG.debug("{}: Checkpoint {} canceled, aborting alignment.", taskName, cancelledId);
-		}
```

Comment:
Have we lost this log message?

Reply:
I guess this log message is not always valuable and the message `aborting alignment` might be misleading especially in the case of cancellation id less than the current checkpoint id, because we actually ignore it in that case. I will consider add this debug log under the below condition.